support-frontend/assets/helpers/page/page.ts

```diff
@@ -1,19 +1,15 @@
 // ----- Imports ----- //
 import type { Participations } from 'helpers/abTests/abtest';
-import * as abTest from 'helpers/abTests/abtest';
-import { Country, CountryGroup } from 'helpers/internationalisation';
+import { Country } from 'helpers/internationalisation';
 import type { IsoCountry } from 'helpers/internationalisation/country';
-import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
 import {
 	analyticsInitialisation,
 	consentInitialisation,
 } from 'helpers/page/analyticsAndConsent';
 import { getReferrerAcquisitionData } from 'helpers/tracking/acquisitions';
 
-function setUpTrackingAndConsents(): void {
+function setUpTrackingAndConsents(participations: Participations = {}): void {
 	const countryId: IsoCountry = Country.detect();
-	const countryGroupId: CountryGroupId = CountryGroup.detect();
-	const participations: Participations = abTest.init(countryId, countryGroupId);
```

Comment:
Does this potentially break [all the other places that use this](https://github.com/search?q=repo%3Aguardian%2Fsupport-frontend+%22import+%7B+setUpTrackingAndConsents+%7D+from+%27helpers%2Fpage%2Fpage%27%3B%22&type=code)?

Reply:
We thought to have a follow up PR to update setUpTrackingAndConsents() and pass participations to that in places where state is available.Right now , we thought setting a default value for participations in setUpTrackingAndConsents()  will help to not break other parts of code

Reply:
@jamesgorrie is correct it will break other instances where we call `setUpTrackingAndConsents()` unless we pass `participations`.

This branch has 2 commits. I think the 1st commit in this branch didn't have this issue, as all the logic to fix the issue was in `setUpTrackingAndConsent`. 

However, it looks like the 2nd commit introduces the issue @jamesgorrie highlighted. I think the first commit should be the change to be merged here and we should revert the 2nd commit - as this was a 2nd iteration, which needs further development?

Reply:
@jamesgorrie @LAKSHMIRPILLAI I've reverted the 2nd commit in the PR, which means this PR doesn't alter the function signature for `setUpTrackingAndConsents()`, so it shouldn't impact places the function is called.